_game/shines/b3.md

```diff
@@ -6,26 +6,46 @@ parent: Shines
 nav_order: 6
 ---
 # Bianco 3
-
+Bianco 3 is made up of two parts: the Entry and the Secret. You should be comfortable with hoverslides, (jump) dive rollouts, spin jumps, sideflips, and triple jumps to have success with this shine.
 ## Entry
-
+Both the Beginner and Advanced Entry strats utilize a majority of the same movement. The one difference will be called out in the Advanced section.
 ### Beginner
-- Cliff Wall Kick
-### Intermediate
-- Pole Wall Kick
+#### Pole Wall Kick
+- From the start, hold up and hoverslide
+- Aim towards roughly the middle of the log and buffer a hover by holding R
+- Briefly hover when Mario slides off the path and double jump off the log
+- Depending on the speed retained from the double jump, you may need to dive rollout to reach the doorway
+- Dive through the doorway, rollout, then jump and spray up the path. Dive into the water left on the path
+- Slide to the water, rollout onto the first log, jump dive to the second log
+- Rollout onto the land, dive rollout up the small hill to position yourself to the right of the pole
+- Spin jump or sideflip into the pole to begin the wall kick
+- Aim to the [right side of the pole](https://cdn.discordapp.com/attachments/941086197885833266/1029200667119730708/unknown.png) as Mario can only wall kick off the right side.
```

Comment:
For this, and other images, you can use the following structure to embed them inline:
`![](image link)`

Up to you if you think that looks good or not! (goes for all images you're linking here)